### PR TITLE
Docs facelift

### PR DESCRIPTION
**Pull request by tylerhutcherson**
- Reorganized docs structure with better headers
- Added more images
- Added code snippets
- Added integration guide
- Edited copy to be more web-readible

### CODE REVIEW

**Comment by eecaro** [comment]
@tylerhutcherson is it intended to have a `View on Github` button link to the documentation repo specifically?

Screen Shot 2019-07-17 at 10 51 06 AM

File: index.md [hunk 1]
```diff
@@ -1,47 +1,123 @@
...
+## [Dashboard](https://dashboard.skafos.ai)
+A web-based user interface designed to help you do the following:
 
-## Have questions? Need help?
-Read through our set of [FAQs](sections/faq.md) (Frequently Asked Questions).
+#### Create New App Integrations
+
+<img src="assets/new_app.png" width="40%"/>
+
+  - Start with an example app
```
Comment (eecaro) [inline index.md]: Would it be useful to include the iOS example links from the resources section or list the available types of example apps ready to be used?

File: sections/integrate.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+In order to receive model updates over the air, embed the `Skafos.load` method in your
+view controller and change `MyCustomModel()` to match your CoreML model name:
+
+```swift
+import UIKit
+import Skafos
+
+class MainViewController : UIViewController {
+    // This is the model name you assigned on the dashboard
+    private let modelName:String = "<your-skafos-model-name>"
```
Comment (eecaro) [inline sections/integrate.md]: Would it be useful to also mention in the short text summary above (line 71 and 72) that the `modelName` should also be updated to match the name of your model in the dashboard?

File: sections/integrate.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            }
+        }
+    }
+}
+```
+
+----
+
+*You must have an initial model version in your app bundle. We highly recommend using CoreML (.mlmodel),
+but other formats are possible. [See some of our example apps delivering non-CoreML models](https://github.com/skafos/example-ml-apps)*
```
Comment (eecaro) [inline sections/integrate.md]: Should this be its own section like the others above? It feels like it has the potential to be skipped by someone skimming our documentation. Have we had Skafos users bring up in our support slack regarding this issue?

File: sections/integrate.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+---
+title: Skafos Integration
```
Comment (eecaro) [inline sections/integrate.md]: Would it be possible to have the links to the integration guide open in a new tab or way to navigate back to the "home" page of the docs in the header?

File: sections/faq.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Frequently Asked Questions
```
Comment (eecaro) [inline sections/faq.md]: Would it be possible to have the links to the FAQ open in a new tab or way to navigate back to the "home" page of the docs in the header?

File: index.md (hunk 1 above)
Comment (tylerhutcherson) [reply index.md]: feels like it's clearer to keep all links to external examples in one spot so we aren't sending them around places before they've read through or understood more of the platform, just my perspective

File: sections/integrate.md (hunk 2 above)
Comment (tylerhutcherson) [reply sections/integrate.md]: Will do.

File: sections/integrate.md (hunk 3 above)
Comment (tylerhutcherson) [reply sections/integrate.md]: Done!

File: sections/integrate.md (hunk 4 above)
Comment (tylerhutcherson) [reply sections/integrate.md]: Yep

File: sections/faq.md (hunk 5 above)
Comment (tylerhutcherson) [reply sections/faq.md]: Indeed

**Pull request merged**